Main.java

```diff
@@ -9,7 +10,38 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // scanner for user input 
+    Scanner input = new Scanner(System.in);
+
+    //asks user for input
+    System.out.println("Please enter a number to play FizzBuzz");
+    
+    //user input
+    int number = input.nextInt();
+
+    int Fizz = (number%3);
+    int Buzz = (number%5);
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for.